tests/integration/helm/helm-internal-auth_test.go

```diff
@@ -204,8 +210,7 @@ func (s *internalAuthHelmTest) TestHelmInstall() {
 
 		s.Run(testCase.name, func() {
 			subT := s.T()
-			// Disabling parallelization until we configure discrete databases
-			// subT.Parallel()
+			subT.Parallel()
```

Comment:
If we are running the tests now in parallel, should we create a make target that calls `terratest_log_parser` like `test-integration-helm-interleaved-internal`? Else parsing the logs may become difficult.

Reply:
If you think that's helpful, then let's do it. I ran `terratest_log_parser` locally and it wasn't extremely helpful for me, however, if the team thinks it's helpful, I say let's do it.

Reply:
Added a make recipe for interleaved